inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarUtils.java

```diff
@@ -39,54 +51,420 @@ public class PulsarUtils {
     private PulsarUtils() {
     }
 
+    public static final String QUERY_CLUSTERS_PATH = "/admin/v2/clusters";
+    public static final String QUERY_BROKERS_PATH = "/admin/v2/brokers";
+    public static final String QUERY_TENANTS_PATH = "/admin/v2/tenants";
+    public static final String QUERY_NAMESPACE_PATH = "/admin/v2/namespaces";
+    public static final String QUERY_PERSISTENT_PATH = "/admin/v2/persistent";
+    public static final String LOOKUP_TOPIC_PATH = "/lookup/v2/topic";
+
+    private static final Gson GSON = new GsonBuilder().create(); // thread safe
+
+    /**
+     * Get http headers by token.
+     *
+     * @param token pulsar token info
+     * @return add http headers for token info
+     */
+    private static HttpHeaders getHttpHeaders(String token) {
+        HttpHeaders headers = new HttpHeaders();
+        if (StringUtils.isNotEmpty(token)) {
+            headers.add("Authorization", "Bearer " + token);
+        }
+        return headers;
+    }
+
     /**
-     * Get pulsar admin info
+     * Get pulsar cluster info list.
+     *
+     * @param restTemplate spring framework RestTemplate
+     * @param clusterInfo pulsar cluster info
+     * @return list of pulsar cluster infos
+     * @throws Exception any exception if occurred
      */
-    public static PulsarAdmin getPulsarAdmin(PulsarClusterInfo pulsarCluster) throws PulsarClientException {
-        Preconditions.expectNotBlank(pulsarCluster.getAdminUrl(), ErrorCodeEnum.INVALID_PARAMETER,
-                "Pulsar adminUrl cannot be empty");
-        PulsarAdmin pulsarAdmin;
-        if (StringUtils.isEmpty(pulsarCluster.getToken())) {
-            pulsarAdmin = getPulsarAdmin(pulsarCluster.getAdminUrl());
+    public static List<String> getPulsarClusters(RestTemplate restTemplate, PulsarClusterInfo clusterInfo)
+            throws Exception {
+        final String url = clusterInfo.getAdminUrl() + QUERY_CLUSTERS_PATH;
+        return HttpUtils.request(restTemplate, url, HttpMethod.GET, null, getHttpHeaders(clusterInfo.getToken()),
+                ArrayList.class);
+    }
+
+    /**
+     * Get the list of active brokers.
+     *
+     * @param restTemplate spring framework RestTemplate
+     * @param clusterInfo pulsar cluster info
+     * @return list of pulsar broker infos
+     * @throws Exception any exception if occurred
+     */
+    public static List<String> getPulsarBrokers(RestTemplate restTemplate, PulsarClusterInfo clusterInfo)
+            throws Exception {
+        final String url = clusterInfo.getAdminUrl() + QUERY_BROKERS_PATH;
```

Comment:
url should contains cluster name, like `http://localhost:8080/admin/v2/brokers/$clusterName`